validate-cert.js

```diff
@@ -12,12 +12,19 @@ module.exports = function validate(pem_cert) {
       return 'subjectAltName Check Failed'
     }
 
+    // check that the signing certificate has not expired (examine Not After)
+    var now = new Date().getTime()
     var notAfter = new Date(cert.validity.notAfter)
-    var remainingDays = notAfter.getTime() - new Date().getTime()
-    // check that the signing certificate has not expired (examine both the Not
-    // Before and Not After dates)
-    if (remainingDays < 1) {
-      return 'certificate expiration check failed'
+    var timeUntilAfter = notAfter.getTime() - now
+    if (timeUntilAfter < 1) {
+      return 'certificate Not After check failed'
```

Comment:
this logic is kind of hard to read...I'm not sure if this check is validating the end date or start date of the given certificate. can you simplify/rename what is happening here so that's more clear?